README.md

```diff
@@ -1,6 +1,5 @@
-[![CircleCI](https://circleci.com/gh/hashicorp/consul-terraform-sync/tree/main.svg?style=svg&circle-token=a88491ffa8b02149fc483c29c6b8b91ed771f5a5)](https://circleci.com/gh/hashicorp/consul-terraform-sync/tree/main)
+# Consul-Terraform-Sync [![Build Status](https://github.com/hashicorp/consul-terraform-sync/actions/workflows/build.yml/badge.svg)](https://github.com/hashicorp/consul-terraform-sync/actions/workflows/build.yml)
```

Comment:
We still use CircleCI to run our tests, so could the build status be added in addition to the CircleCI rather than replacing it?

Reply:
Hey @mdeggies! Wanted to follow up to get your thoughts on keeping the CircleCi in addition to the GHA badge?

Reply:
So sorry I missed these github notifications, and thank you for the extra ping to bring this to my attention! I updated this to include both the github action build status, and the circleci tests. It looks like this in a markdown editor online: 

<img width="777" alt="Screen Shot 2022-04-08 at 3 50 40 PM" src="https://user-images.githubusercontent.com/12939567/162543815-30c51b40-8c9f-4965-b3c6-1ef0ef15e527.png">

Reply:
Let me know if this looks okay or if there are any other changes to make. 👍